odoo/addons/base/models/ir_sequence.py

```diff
@@ -97,6 +98,25 @@ class IrSequence(models.Model):
     _order = 'name'
     _allow_sudo_commands = False
 
+    _BASE_LEGENDS = {
```

Comment:
```suggestion
    BASE_LEGENDS = {
```
move out of class same for all static